source/testing/index.md

```diff
@@ -19,27 +19,28 @@ the example scenario above, some acceptance tests you might write are:
 
 ### Unit Tests
 
-Unit tests are used to test isolated chunks of functionality, or "units", without 
-worrying about their dependencies. Some examples of unit tests for the scenario 
+Unit tests are used to test isolated chunks of functionality, or "units", without
+worrying about their dependencies. Some examples of unit tests for the scenario
 above might be:
 
 * A user has a role
 * A user has a username
-* A user has a fullname attribute which is the aggregate of its first and last 
+* A user has a fullname attribute which is the aggregate of its first and last
   names with a space between
 * A post has a title
 * A post's title must be no longer than 50 characters
 
 ### Testing Frameworks
 
-[QUnit] is the default testing framework for this guide, but others are 
+[QUnit] is the default testing framework for this guide, but others are
 supported through third-party addons.
 
 ### How to Run Your Tests
 
-Run your tests with `ember test` on the command-line. You can re-run your tests on every file-change with `ember test --server`. For more details and options, see [Ember CLI – Testing](http://www.ember-cli.com/user-guide/#testing) and `ember help test`.
+Run your tests with `ember test` on the command-line. You can re-run your tests on every file-change with `ember test --server`. These commands run your tests using [Testem] to make testing multiple browsers very easy. You can configure Testem using the `testem.json` file in your application root.
+
+Alternatively you can run the tests in your regular browser using the QUnit interface. Run `ember server` and navigate to `http://localhost:4200/tests`. When the app runs in /tests it runs in the development environment, not the test environment.
```

Comment:
We've been talking about trying to simplify the guides to just talk about the recommended happiest path. In that spirit, I'd suggest we take out this paragraph.